debian/control

```diff
@@ -0,0 +1,15 @@
+Source: neofs-node
+Section: neofs
+Priority: optional
+Maintainer: NeoSPCC <tech@nspcc.ru>
+Build-Depends: debhelper-compat (= 13), git, bash-completion, dh-sequence-bash-completion
```

Comment:
Are `bash-completion` and `git` build dependencies?

Reply:
I'll remove bash-completion, it's duplicated by dh-sequence-bash-completion
Git is used in Makefile

Reply:
> Git is used in Makefile

git is not required

Reply:
Git is required at least for correct version generation

Reply:
we have `VERSION` file for that. do we need to have a precise version in that context?

Reply:
Package can be built for each commit, not only for releases with updated VERSION file.

Reply:
Now version creation for package will mimic the Makefile logic so using VERSION as fallback method if git isd not present.
@532910 @carpawell

Reply:
> it's not "any" golang I afraid it needs and too many systems have it installed manually :-(

We need some `golang` there anyway, it won't build without it obviously.

Reply:
`dh-sysuser` is missing as well.

Reply:
> > it's not "any" golang I afraid it needs and too many systems have it installed manually :-(
> 
> We need some `golang` there anyway, it won't build without it obviously.

@roman-khimov sorry my build system has Go installed via "wget https://dl.google.com/go/go1.18.4.linux-amd64.tar.gz"
Not having Go package should not break the build - Go seems to be the only exception fortunately

Reply:
> `dh-sysuser` is missing as well.

not used

Reply:
>  Not having Go package should not break the build - Go seems to be the only exception fortunately

This breaks normal Debian source packages, the expectation is that I can take an `*orig.tar.gz` and a `*debian.tar.gz`, install `Build-Depends` and be able to build the package.

>>  dh-sysuser is missing as well.
>
> not used

Yep, an updated version doesn't need it.

Reply:
>This breaks normal Debian source packages, the expectation is that I can take an *orig.tar.gz and a *debian.tar.gz, install Build-Depends and be able to build the package.
Well this project does not have *orig.tar.gz anyways?

Reply:
I think it does, https://github.com/nspcc-dev/neofs-node/releases/tag/v0.32.0

Reply:
Sorry, let's start again.
Target for the package is Debian 11 as stable.
This project can't be built on Debian 11 with Go packages provided with distro.
Removing the dependency seems to be an easy way to avoid this problem.